engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/runner/FlinkProcessMain.scala

```diff
@@ -30,8 +29,8 @@ trait FlinkProcessMain[Env] extends FlinkRunner with LazyLogging {
       runProcess(env, modelData, process, processVersion, ExecutionConfigPreparer.defaultChain(modelData, buildInfo))
     } catch {
       // marker exception for graph optimalization
-      case ex: ProgramAbortException =>
-        throw ex
+      //case ex: ProgramAbortException =>
```

Comment:
Is it intended?

Reply:
Yes, in Flink 1.11 it works differently, I added not-so-nice fallback